src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -45,33 +50,61 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
   const hasErrors = typeof errors !== 'undefined';
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
     const changedVal = e.currentTarget.value;
     setTouched?.(true);
     onChangeHandler(changedVal);
     onValidateHandler?.(setErrors, changedVal);
   };
 
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const handleMouseDownPassword = (e: ClickEvent) => {
+    e.preventDefault();
+  };
+
+  const isConfirmPassword = name === 'confirmPassword';
```

Comment:
If you refactor the aria-label part below (moving to `src/transformer/i18n/transformInputPassword.ts`) then this can be removed.